Bug: Wunused-const-variable when building assimp.

**Describe the bug**
A clear and concise description of what the bug is.

ASSIMP master generates errors on latest version of LLVM: 
Compile with llvm16 master, try to compile. Compilation will stop with the following error:
```
/Users/agokaslan/git/my_forks/assimp/tools/assimp_cmd/Info.cpp:58:16: error: unused variable 'TREE_BRANCH_ASCII' [-Werror,-Wunused-const-variable]
constexpr char TREE_BRANCH_ASCII[] = "|-";
               ^
/Users/agokaslan/git/my_forks/assimp/tools/assimp_cmd/Info.cpp:60:16: error: unused variable 'TREE_STOP_ASCII' [-Werror,-Wunused-const-variable]
constexpr char TREE_STOP_ASCII[] = "'-";
               ^
/Users/agokaslan/git/my_forks/assimp/tools/assimp_cmd/Info.cpp:62:16: error: unused variable 'TREE_CONTINUE_ASCII' [-Werror,-Wunused-const-variable]
constexpr char TREE_CONTINUE_ASCII[] = "| ";
               ^
3 errors generated.
```

**Expected behavior**
It should compile.

**Platform (please complete the following information):**
 - OS: OSX
 - Browser NA
 - Version Master

--- a/tools/assimp_cmd/Info.cpp
+++ b/tools/assimp_cmd/Info.cpp
@@ -55,26 +55,28 @@ constexpr char AICMD_MSG_INFO_HELP_E[] =
         "\t-v,--verbose: Print verbose info such as node transform data\n"
         "\t-s, --silent: Print only minimal info\n";
 
-constexpr char TREE_BRANCH_ASCII[] = "|-";
-constexpr char TREE_BRANCH_UTF8[] = "\xe2\x94\x9c\xe2\x95\xb4";
-constexpr char TREE_STOP_ASCII[] = "'-";
-constexpr char TREE_STOP_UTF8[] = "\xe2\x94\x94\xe2\x95\xb4";
-constexpr char TREE_CONTINUE_ASCII[] = "| ";
-constexpr char TREE_CONTINUE_UTF8[] = "\xe2\x94\x82 ";
 
 // note: by default this is using utf-8 text.
 // this is well supported on pretty much any linux terminal.
 // if this causes problems on some platform,
 // put an #ifdef to use the ascii version for that platform.
 #ifdef _WIN32
+constexpr char TREE_BRANCH_ASCII[] = "|-";
+constexpr char TREE_STOP_ASCII[] = "'-";
+constexpr char TREE_CONTINUE_ASCII[] = "| ";
+
 const char *TREE_BRANCH = TREE_BRANCH_ASCII;
 const char *TREE_STOP = TREE_STOP_ASCII;
 const char *TREE_CONTINUE = TREE_CONTINUE_ASCII;
-#else
+#else  // _WIN32
+constexpr char TREE_BRANCH_UTF8[] = "\xe2\x94\x9c\xe2\x95\xb4";
+constexpr char TREE_STOP_UTF8[] = "\xe2\x94\x94\xe2\x95\xb4";
+constexpr char TREE_CONTINUE_UTF8[] = "\xe2\x94\x82 ";
+
 const char *TREE_BRANCH = TREE_BRANCH_UTF8;
 const char *TREE_STOP = TREE_STOP_UTF8;
 const char *TREE_CONTINUE = TREE_CONTINUE_UTF8;
-#endif
+#endif  // _WIN32
 
 // -----------------------------------------------------------------------------------
 unsigned int CountNodes(const aiNode *root) {

